Integrations/Gmail/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+Added 6 new commands:
```

Comment:
fix file format as specified in the docs(https://github.com/demisto/internal-content/tree/master/documentation/change-log)